client/webpack.config.node.js

```diff
@@ -52,13 +53,22 @@ function getExternals() {
 		// with modules that are incompatible with webpack bundling.
 		nodeExternals( {
 			allowlist: [
-				// `@automattic/components` is forced to be webpack-ed because it has SCSS and other
-				// non-JS asset imports that couldn't be processed by Node.js at runtime.
-				'@automattic/components',
-
-				// The polyfills module is transpiled by Babel and only the `core-js` modules that are
-				// needed by current Node.js are included instead of the whole package.
+				'@automattic/calypso-analytics',
```

Comment:
Can we generate this list automatically?

Reply:
Yes, that would be great. Instead of a static list, there could be a function `isMonorepoPackage` that determines the monorepo-ness of a package. But I don't know how to write such a test right now.

Reply:
You could find all `package.json` in `packages/*/package.json`, read the name and compose this list dynamically.

Otherwise we risk anybody adding a new package used in client or server and breaking only in production.

Reply:
Nice idea, implemented using `globby` and it seems to work beautifully :+1: